packages/node/src/behaviors/access-control/AccessControlServer.ts

```diff
@@ -57,20 +64,35 @@ export class AccessControlServer extends AccessControlBehavior.with("Extension")
     }
 
     #online() {
-        // Handle Backward compatibility to Matter.js before 0.9.1 and add the missing ACL entry if no entry was set
-        // so far by the controller
-        const fabrics = this.env.get(FabricManager);
         const acl = deepCopy(this.state.acl);
         const originalAclLength = acl.length;
+
+        // Collect ACLs for each fabric
+        const aclsForFabric = new Map<FabricIndex, AccessControlTypes.AccessControlEntry[]>();
```

Comment:
This block appears to be duplicated in `#updateAllAclsForFabric`, maybe make a utility function?